src/relay/pass/quantize.cc

```diff
@@ -329,9 +329,11 @@ float ChooseDomScale(const std::vector<const QRealizeIntExprNode*>& nptrs) {
 
 
 /* \brief Unify the dom scale of arguments */
-Array<Expr> UnifyDTypeScale(const Array<Expr>& args,
+Array<Expr> UnifyDTypeScale(const Array<Expr>& ref_args,
```

Comment:
It would be better to decouple `stop_fusion` and `unify_dtype_scale`. Can we implement another simple visitor for `stop_fusion`? Also, since `stop_fusion` is required by some hardwares, we can have a config for turning on/off it. Also would like to here @tqchen 's opinion

Reply:
We can have an optional flag to turn it on and off, however the decision seems to be really baked into the quantization, so it is fine to have it in a single rule